libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -887,19 +887,21 @@ void AC_AttitudeControl::reset_target_and_rate()
 }
 
```

Comment:
reset_target_and_rates() method above should also have an option to not reset the rates so it can be used in acro flight mode for heli.